addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -66,7 +69,19 @@ public WebserviceHandler(Bridge bridge) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no code needed.
+        try {
+            if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {
+                logger.debug("HandleCommand recieved: {}", channelUID.getId());
+                if (command.toString() == "OFF") {
```

Comment:
I have a new problem,
I changed "if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {"
to
"if (channelUID.getId()==CHANNEL_HOLIDAY)" {"
although the watch shows for both sides "holiday" the result is False!???